data/CBID2.json

```
@@ -53,12 +45,20 @@
       "url": "https://www.linkedin.com/posts/christinebelzie_announcing-our-new-in-tech-scholarship-winner-activity-6993952117304508416-3oC5"
     },
     {
-      "title": "Blog post got chosen as the Community Pick on daily.dev",
-      "date": "December 2022",
+      "title": "Accepted into FreeCodeCamp's Writing Team",
+      "date": "April 2023",
+      "icon": "fa-free-code-camp",
```

Here you probably be using an invalid icon. It should be `FaFreeCodeCamp`.

```suggestion
      "icon": "FaFreeCodeCamp",
```